CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# vNext
```

Comment:
Just add these to 7.0.0

Reply:
I usually pin the SDK for reproducible builds, but since in CI the dotnet install script should install the one specified in the global.json anyway, I can add it back

Reply:
Also I am adding a new CI, which does not require sonar. It seems like that in the last few years github changed the workflow system, so that forks can't access secrets :/

Reply:
So, hopefully now everything should work for PRs. Can you update this one too so the new ci is run?

Also, sorry for the problems around prs-I have not touched this repo in two years, and it seems I did not really make it future proof